manifests/cloner/script.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ "$1" == "source" ] ; then
+  mkfifo /tmp/clone/socket/pipe; cat /tmp/clone/image/disk.img > /tmp/clone/socket/pipe
+elif [ "$1" == "target" ] ; then
+  while true; 
```

Comment:
I recommend using a limited number of retries and once exhausted exit with a timeout error.

Reply:
Done

Reply:
how is the failure communicated (if the `while` loop is ended)?